core/src/test/java/org/elasticsearch/search/builder/SearchSourceBuilderTests.java

```diff
@@ -663,4 +663,25 @@ public void testEmptyQuery() throws IOException {
         String query = "{ \"query\": {} }";
         assertParseSearchSource(builder, new BytesArray(query), ParseFieldMatcher.EMPTY);
     }
+
+    public void testSearchRequestBuilderSerializationWithIndexBoost() throws Exception {
```

Comment:
It'd be fine with me if it were one of the random options for `createSearchSourceBuilder`, yeah. We'd want to make sure we kept the assertion that the deserialized copy's bytes match the original's bytes which I don't think is an assertion we normally have in these round trip tests.